WPMedia/fusion-news-theme-blocks
# Updated content api to be searchable in PB Editor

**Be sure to run `npm test`, `npm run lint`, and write detailed test steps before requesting reviewers**

## Description

#### Jira Ticket: [THEMES-1085](https://arcpublishing.atlassian.net/browse/THEMES-1085)
#### Jira Ticket: [THEMES-1267](https://arcpublishing.atlassian.net/browse/THEMES-1267)
#### Jira Ticket: [THEMES-1268](https://arcpublishing.atlassian.net/browse/THEMES-1268)

Updated the content-api block to allow for searchable stories.

## Test Steps

_Add detailed test steps a reviewer must complete to test this PR_

1. Checkout this branch `git checkout THEMES-1085`
2. Run fusion repo with linked blocks `npx fusion start -f -l @wpmedia/content-api-source-block`
3. Open a page in Pagebuilder, and add a small, medium, large, and xl promo block.
4. In each block, under 'Display Content Info' select 'ans-item', and under 'Content Source' select 'content-api'
5. Verify that the "Select Story" button appears, and select a story
6. Verify that the block successfully renders the selected story.

<img width="339" alt="image" src="https://github.com/WPMedia/arc-themes-blocks/assets/85515364/554b3e2f-4eb7-497c-aa8f-695230cca35e">


[THEMES-1085]: https://arcpublishing.atlassian.net/browse/THEMES-1085?atlOrigin=eyJpIjoiNWRkNTljNzYxNjVmNDY3MDlhMDU5Y2ZhYzA5YTRkZjUiLCJwIjoiZ2l0aHViLWNvbS1KU1cifQ

[THEMES-1267]: https://arcpublishing.atlassian.net/browse/THEMES-1267?atlOrigin=eyJpIjoiNWRkNTljNzYxNjVmNDY3MDlhMDU5Y2ZhYzA5YTRkZjUiLCJwIjoiZ2l0aHViLWNvbS1KU1cifQ
[THEMES-1268]: https://arcpublishing.atlassian.net/browse/THEMES-1268?atlOrigin=eyJpIjoiNWRkNTljNzYxNjVmNDY3MDlhMDU5Y2ZhYzA5YTRkZjUiLCJwIjoiZ2l0aHViLWNvbS1KU1cifQ
